test/qa-tests/jstests/top/mongotop_sharded.js

```diff
@@ -11,7 +11,7 @@ var testName = 'mongotop_sharded';
     assert.strContains.soon(expectedError, getOutput, 'error message must appear at least once');
     var shellOutput = getOutput();
     jsTest.log('shell output: ' + shellOutput);
-    shellOutput.split('\n').forEach(function(line) {
+    shellOutput.split('\n').slice(1).forEach(function(line) {
       // check the displayed error message
       assert.neq(line.match(expectedError), null, 'unexpeced error message');
```

Comment:
[nit] typo in "unexpected"

Reply:
ah, Thanks!

Reply:
I will change this before the merge. Don't wanna rerun the evergreen :-)